sample_files/sample_CSV/test_grades.py

```diff
@@ -0,0 +1,61 @@
+# import requests
+# from requests.auth import HTTPBasicAuth
+from verify import parse_assigned_zones, get_actual_zone_dict
+
+actual_dict = get_actual_zone_dict()
+assigned_dict = parse_assigned_zones()
+
+output_list = []
+file_list = ['TestExam_1']
+for file in file_list:
+    with open('%s.csv' % (file, ), 'r') as zones:
+        # Get header row contents
+        header_str = zones.readline()
+
+        line_list = zones.readlines()
```

Comment:
When parsing CSV files I recommend using the built in CSV library. Right now this will have issues if any field has a comma as part of the value. (For example, comments might contain something like `"Late, had excuse"`, or zones might be something like `"HOUSTON35,10"`). I'd recommend using [Python's csv library](https://docs.python.org/3/library/csv.html) to avoid having fragile parsing. 

It'll also reduce the next few lines into one `for ____ in csvreader:` line